tests/check_objects_and_errors.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+echo "Running objects.go to update files"
+pushd crypto/obj
+go run objects.go
+popd
+
+echo "Adding all changed files to the git tree"
+git add -A
+
+echo "Checking for any changes"
+git diff --exit-code HEAD
+
+echo "Running make_errors.go to update files"
+go run ./util/make_errors.go ssl evp ocsp
```

Comment:
I don't see a handling of the `ocsp` argument in make_errors.go. What am I missing?

Reply:
It's passed to find the corresponding header file, which defines the error macro.

This is a good question and we may need to check more header files.

Reply:
`ocsp` doesn't need special case handling, while the others do. It processes the `ocsp.h` header file here.
https://github.com/awslabs/aws-lc/blob/35255bdb05eab786a7f184fe774dc798d40f32e6/util/make_errors.go#L53

@bryce-shang has synced the `ocsp` error data before with this go file, so it's confirmed to work: https://github.com/awslabs/aws-lc/commit/7a0c2846743ebd54a76127feeae2b243873df2c1